### PR TITLE
Add testnets to selector

### LINKED ISSUE #55: Add testnets from the registry
@samepant any chance we could include testnets as well for testing this tool? My accountant would appreciate it if I could not use real ATOMs ;)

I've updated my chain registry repo to include testnets, I think you can probably copy a bunch: https://github.com/nooomski/registry-ui

### PR DESCRIPTION
**Pull request by abefernan**
Closes #55

### CODE REVIEW

File: components/chainSelect/ChainSelect.tsx [hunk 1]
```diff
@@ -61,20 +61,28 @@ const ChainSelect = () => {
...
-      const res = await axios.get(url);
-      const chains = res.data.filter((item: GithubChainRegistryItem) => {
-        return item.type == "dir" && !item.name.startsWith(".") && item.name != "testnets";
-      });
-      setChainArray(chains);
-      const options = chains.map(({ name }: GithubChainRegistryItem, index: number) => {
-        return { label: name, value: index };
-      });
+      const { data: chains } = await axios.get(chainsUrl);
+      const { data: testnets } = await axios.get(testnetsUrl);
```
Comment (webmaster128) [inline components/chainSelect/ChainSelect.tsx:71]: Let's download them in parallel with `Promise.all([axios.get(chainsUrl), axios.get(testnetsUrl)])`

**Review by webmaster128 — COMMENTED** [review]
Can we separate the testnet entries in the list somehow? Maybe add a -------- separator and then list all testnets in the bottom?

**Comment by abefernan** [comment]
> Can we separate the testnet entries in the list somehow? Maybe add a -------- separator and then list all testnets in the bottom?

I got into a pretty buggy rabbit hole trying to do this, maybe we can extract that UX improvement to a different ticket?

**Comment by webmaster128** [comment]
> I got into a pretty buggy rabbit hole trying to do this, maybe we can extract that UX improvement to a different ticket?

Sure! One step at a time

**Review by webmaster128 — APPROVED** [review]

**Pull request merged**